src/box/applier.cc

```diff
@@ -2227,6 +2227,13 @@ applier_f(va_list ap)
 				goto reconnect;
 			} else {
 				/* Unrecoverable errors */
+				ERROR_INJECT(
+					ERRINJ_APPLIER_FAIL_DESTROY_DELAY, {
+					say_warn("applier fail delay destroy");
```

Comment:
If we don't want to grep log, there's an alternative: we may also create incrementing error injection: 0 - nothing happened, 1 - yielding started, 2 - yielding ended, But in general it will be just the same, just no log output and inability to use `ERROR_INJECT_YIELD` macros

Reply:
grepping the log seems just fine IMO.

Reply:
Yep, seems fine for me too